README.md

```diff
@@ -32,7 +32,10 @@ $ make build
 
 Using the provider
 ----------------------
-__TODO__: Fill in for each provider
+
+If you're building the provider, follow the instructions to [install it as a plugin.](https://www.terraform.io/docs/plugins/basics.html#installing-a-plugin) After placing it into your plugins directory,  run `terraform init` to initialize it.
```

Comment:
Is this part aimed at folks developing the provider? I don't think that regular users will need anything other than `terraform init` since it's an official provider

Reply:
Agreed this feels like something that would occur during development of the provider itself. Should this be moved to the `Developing the Provider` section?